cuebot/build.gradle

```diff
@@ -93,12 +95,7 @@ sourceSets {
     }
 }
 
-jar {
-    manifest {
-        attributes 'Main-Class': 'com.imageworks.spcue.CuebotApplication'
-    }
+bootJar {
```

Comment:
Could you explain the difference here?

Reply:
Spring boot has its own and superior to anything else jar packager.